### DISCUSSION TITLE
Things to do before creating the pull request

### DISCUSSION THREAD

**Discussion by yashh-dev** [h2oai/wave#1772] [Q&A] [answered]
- Do i need to run `make generate` before pushing to repo.
- Do i need to Generate assets for website docs before pushing

**Comment by mturoci** [comment, marked as answer]
Depends on the nature of the change. In general, reviewers will guide you in case there is something else that needs to be done.

If you change/add TS API, then you need to run `make generate` to generate python/R API. As for the assets, these need to be added only if new example was added to the Tour app.

**Reply by yashh-dev** [reply]
thanks for the response
